packages/core/test/constants.test.ts

```diff
@@ -6,7 +6,7 @@ describe("core/constants.ts", () => {
   })
 
   test("Use staging endpoint URLs when not in production", () => {
-    expect(DEFAULT_SERVICE_ENDPOINT).toBe("https://ads.allizom.org/")
+    expect(DEFAULT_SERVICE_ENDPOINT).toBe("https://ads.mozilla.org/")
```

Comment:
This isn't testing the right thing anymore.

Reply:
Good point, thank you so much. Will fix.